src/src/rx_main.cpp

```diff
@@ -505,8 +526,9 @@ uint32_t ICACHE_RAM_ATTR HandleFreqCorr(bool value)
         {
             tempFC--; // FREQ_STEP units
         }
-        else
+        else if (tempFC == FreqCorrectionMin)
         {
+            tempFC--;
             DBGLN("Max -FreqCorrection reached!");
         }
```

Comment:
This seems like the wrong way. I assume you are wanting the message only printed when it reaches the limit and not spam continually when it's at the limit.
If so then it should really be like this...
```c++
{
  tempFC--; // FREQ_STEP units
  if (tempFC == FreqCorrectionMin)
  {
    DBGLN("Max -FreqCorrection reached!");
  }
}
```
And similar below.
Otherwise the tempFC will go below the Min and over the Max by one.

Reply:
Oh thanks

Reply:
Fixed